src/TicketBai/Mews.Fiscalizations.TicketBai/Model/Invoice/InvoiceHeader.cs

```diff
@@ -0,0 +1,48 @@
+﻿using FuncSharp;
+using Mews.Fiscalizations.Core.Model;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Mews.Fiscalizations.TicketBai.Model
+{
+    public sealed class InvoiceHeader
+    {
+        public InvoiceHeader(
+            String1To20 number,
+            DateTime issueDate,
+            DateTime issueDateTime,
+            bool isSimplified = false,
+            bool issuedInSubstitutionOfSimplifiedInvoice = false,
+            String1To20 series = null,
+            CorrectingInvoice correctingInvoice = null,
```

Comment:
I don't understand. So I'm submitting invoice 20 and I'm already saying that there's gonna be an invoice 21 correcting this stuff?

Reply:
With juts common sense, all I could think of is that CorrectingInvoice  is actually the data about `how much money did we correct with this`. Which would mean it's more like Data and this is the correcting invoice. But hey, what do I know. First time I hear of this. We could try asking someone.

Reply:
Yea exactly, I'm definitely gonna visit this again later.
I will dig deeper into the docs and examples to figure it out.